src/services/releases/docs/index.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+	releasesService: {
+		description: "A service for releases",
+		find: {
+			summary: "find"
+		},
+		create: {
+			summary: "create"
+		},
+		remove: {
+			summary: "remove"
+		},
+		update: {
+			summary: "update"
+		},
+		patch: {
+			summary: "patch"
+		},
+		get: {
+			summary: "get"
+		},
+		releaseFetchService:{
+			find: {
+				parameters: [
```

Comment:
fetch has no parameters, you can just simply fetch the newest releases from GitHub.